### PR TITLE
Appium page object tools. Demonstration. #30 fix

### PR DESCRIPTION
**Pull request by TikhomirovSergey**
Please let me know If here is not enough comments

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
Please review this PR. I think there are lots of interesting thing that want to be known by Appium users :)

**Comment by bootstraponline** [comment]
I like it. :+1:  @Jonahss is the java client maintainer so he should probably review as well.

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenByAllPossible.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import java.util.List;
+
+import io.appium.java_client.android.AndroidElement;
+import io.appium.java_client.pagefactory.AndroidFindAll;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+
+/**
+ * 
+ * @author TikhomirovSergey
```
Comment (bootstraponline) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenByAllPossible.java]: In general we don't use author tags. It's common for many people to contribute using github and adding everyones name to an author tag doesn't work. Git is the best place to record authorship information.

**Comment by TikhomirovSergey** [comment]
@bootstraponline thanks ror the remark.
I have removed __author__  tags and I have added some comments.

**Pull request merged**